src/Temperature-Accretech_UFseries/main.py

```diff
@@ -0,0 +1,160 @@
+# This Device Class is published under the terms of the MIT License.
+# Required Third Party Libraries, which are included in the Device Class
+# package for convenience purposes, may have a different license. You can
+# find those in the corresponding folders or contact the maintainer.
+#
+# MIT License
+#
+# Copyright (c) 2024 SweepMe! GmbH (sweep-me.net)
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# SweepMe! device class
+# Type: Temperature
+# Device: Accretech UF series
+from __future__ import annotations
+
+from pysweepme.EmptyDeviceClass import EmptyDevice
+from pysweepme.FolderManager import addFolderToPATH
+
+addFolderToPATH()
+
+# standard import
+# import accretech_uf
+# importlib.reload(accretech_uf)
+
+# direct import by path
+import imp
+import os
+
+accretech_uf = imp.load_source(
+    "accretech_uf",
```

Comment:
To prevent some cross-talk between different versions of accretech_uf, it would make sense to name it here "accretech_uf_temperature". The variable in the line above does not need to be changed. It is just about the name that is used to register the module in the python system.